### PR TITLE
PhpUnitStrictFixer: handle static calls

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: src/Fixer/PhpUnit/PhpUnitStrictFixer.php [hunk 1]
```diff
@@ -99,26 +99,40 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
...
-                    array(
-                        array(T_VARIABLE, '$this'),
-                        array(T_OBJECT_OPERATOR, '->'),
-                        array(T_STRING, $methodBefore),
-                        '(',
-                    ),
-                    $index
-                );
+            foreach ($sequencesToFind as $sequenceToFind) {
+                for ($index = 0, $limit = $tokens->count(); $index < $limit; ++$index) {
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitStrictFixer.php]: I believe that there is no need to iterate whole tokens collection 3x, originally we did only once.

Comment (Slamdunk) [reply src/Fixer/PhpUnit/PhpUnitStrictFixer.php]: Algorithm optimized

**Comment by keradus** [comment]
`broken pipe` issue again, I restarted travis

**Comment by Slamdunk** [comment]
Rebased to master as for https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3478#issuecomment-365404119

**Comment by keradus** [comment]
Thank you @Slamdunk.

**Pull request merged**